Classes/UNAlertView.swift

```diff
@@ -210,8 +211,17 @@ final public class UNAlertView: UIView {
     func buttonTapped(btn:UNAlertButton) {
         
         btn.action()
+        if (autoDismiss) {
+            dismiss()
+        }
+    }
+    
+    func dismissAlert() {
         dismiss()
     }
+    func setAutoDismiss(autoStatus: Bool) {
+        autoDismiss = autoStatus
+    }
     
```

Comment:
@siberian1967 What's this function for?  I think the variable of autoDismiss is enough for what you want.
```
func dismissAlert() {
    dismiss()
}
```